domains/go.fran.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "xzonix",
+        "email": "imahalojigi@gmail.com"
+    },
+    "record": {
+        "TXT": "cname.short.io"
```

```suggestion
        "CNAME": "cname.short.io"
```
assuming you had to add CNAME instead of TXT